### ISSUE TITLE
Syntax highlighting missing for `let struct`

### ISSUE THREAD

**Issue by MangelMaxime** [ionide/ionide-vscode-fsharp#1937]
### Describe the bug

I feel like `let struct (x0, y0)` should be highlighted the same as `let (One one)`

### Link to sample reproduction
```fs
let origin = struct (0,0)
let (struct (x0,y0)) = origin

type Test =
    | One of int

let value = One 1
let (One one) = value
```

### Screenshots
![CleanShot 2023-09-26 at 22 31 57](https://github.com/ionide/ionide-vscode-fsharp/assets/4760796/8b3412c2-860c-4338-ad51-c3d80f828496)

## Machine info
 - OS: Linux
 - .NET SDK version: [e.g. 5.0.100]
 - Ionide version: 7.12.3

**Comment by jkillingsworth** [comment]
See my comment here:

https://github.com/ionide/ionide-fsgrammar/issues/199#issuecomment-1739651074

**Comment by MangelMaxime** [comment]
You are correct, it should use the `keywoard` color, I updated my initial message.

**Comment by MangelMaxime** [comment]
The grammar has been updated.

What left is the tokens highlighting in FSAC

**Comment by 64J0** [comment]
It seems to be working correctly now.

This is what I'm getting with Ionide 7.29.0:

<img width="270" height="58" alt="Image" src="https://github.com/user-attachments/assets/1554fbd6-d5aa-4957-a5c0-ab1e03a55d90" />

And with different color themes:

<img width="270" height="58" alt="Image" src="https://github.com/user-attachments/assets/db02dcba-d100-4ba7-a631-f6520b0c66e0" />

<img width="270" height="58" alt="Image" src="https://github.com/user-attachments/assets/d9209978-b9a5-42cd-9345-13488537089d" />

<img width="270" height="58" alt="Image" src="https://github.com/user-attachments/assets/78d28a98-54fc-4ea0-8617-4b07f7d1a9fc" />

**Comment by TheAngryByrd** [comment]
thanks for confirming!
